ci/credentials.yml.example

```diff
@@ -95,12 +95,14 @@ aws_external_s3_tfstate_bucket: EXTERNAL_TERRAFORM_BUCKET
 cdn_broker_username_staging: cdn-broker-staging
 cdn_broker_bucket_staging: cdn-broker-le-verify-staging
 cdn_broker_cloudfront_prefix_staging: cg-staging/*
-cdn_broker_hosted_zone_staging:
+cdn_broker_hosted_zone_staging: ZZZZZZZZZZZZZZ
```

Comment:
I have nothing vested in the color of this 🚲  🏚 , but I thought it marginally useful to hint that the value would be a zone id starting with 'Z'. 

Want to merge as is? Or do you prefer an empty string?